src/integrationTest/java/uk/gov/hmcts/reform/cdapi/RetrieveCategoriesIntegrationTest.java

```diff
@@ -30,6 +33,56 @@ public class RetrieveCategoriesIntegrationTest extends CdAuthorizationEnabledInt
 
     private static final String path = "/lov/categories/{category-id}";
 
+    @Test
+    @DisplayName("Retrieve categories with externalRefarence ")
+    void shouldRetrieveCategoriesWithExternalReferenceStatusCode200()
+        throws JsonProcessingException {
+
+        final var response = (Categories)
+            commonDataApiClient.retrieveCaseFlagsByServiceId("panelCategoryMember?serviceId=BBA3",
+                                                                 Categories.class, path
+            );
+        assertNotNull(response);
+        assertEquals(3, response.getListOfCategory().size());
+        assertThat(response.getListOfCategory().get(0).getKey()).isEqualTo("PC1-01-94");
+        assertThat(response.getListOfCategory().get(0).getValueEn()).isEqualTo("Financial office holder");
+        assertThat(response.getListOfCategory().get(0).getExternalReference()).isEqualTo("94");
+        assertThat(response.getListOfCategory().get(0).getExternalReferenceType()).isEqualTo("FinancialRole");
+
+        assertThat(response.getListOfCategory().get(1).getKey()).isEqualTo("PC1-01-84");
+        assertThat(response.getListOfCategory().get(1).getValueEn()).isEqualTo("Judicial office holder");
+        assertThat(response.getListOfCategory().get(1).getExternalReference()).isEqualTo("84");
+        assertThat(response.getListOfCategory().get(1).getExternalReferenceType()).isEqualTo("JudicialRole");
+
+        assertThat(response.getListOfCategory().get(2).getKey()).isEqualTo("PC1-01-74");
+        assertThat(response.getListOfCategory().get(2).getValueEn()).isEqualTo("Medical office holder");
+        assertThat(response.getListOfCategory().get(2).getExternalReference()).isEqualTo("74");
+        assertThat(response.getListOfCategory().get(2).getExternalReferenceType()).isEqualTo("MedicalRole");
+    }
+
+    @Test
+    @DisplayName("Retrieve categories without externalRefarence ")
```

Comment:
Typo, "externalRefarence" should be "externalReference"

Reply:
corrected